src/TestTextSearch.cpp

```diff
@@ -0,0 +1,107 @@
+#include <iostream>
+#include <fstream>
+
+#include "BaseUtil.h"
+// layout controllers

```

Comment:
this is a copy-paste. Granted the comment is misleading in the original place as well.